app/components/modals/new-user/component.js

```diff
@@ -1,7 +1,36 @@
 import Ember from "ember";
+import UniqueFieldValidator from "last-strawberry/validators/unique-field-validator";
+import computed from "ember-computed-decorators";
 
 export default Ember.Component.extend({
+  session:     Ember.inject.service(),
+
+  @computed("emailValidator.isValid", "changeset.isValid")
+  isValid(validEmail, validChangeset) {
+    return validEmail && validChangeset;
+  },
+
+  didInsertElement() {
+    this._super(...arguments);
+
+    this.set("emailValidator", UniqueFieldValidator.create({
+      session:this.get("session"),
+      type:"user",
+      key:"email"}));
+  },
+
+  willDestroyElement() {
+    this._super(...arguments);
+
+    this.get("emailValidator").destroy();
+  },
+
   actions: {
+    emailChanged(newValue) {
+      this.get("changeset").set("email", newValue);
+      this.get("emailValidator").validate(newValue);
```

Comment:
The second argument of type array should be passed with a whitelist of values that should be accepted, this is where we can put the current saved value so that it doesn't make a request and so it also allows the validation to pass. Did this work for you?